extension/src/popup/components/account/AssetDetail/index.tsx

```diff
@@ -183,17 +186,19 @@ export const AssetDetail = ({
           <div className="AssetDetail__actions">
             {balance?.total && new BigNumber(balance?.total).toNumber() > 0 ? (
               <>
-                {/* Hide send for Soroban until send work is ready for Soroban tokens */}
-                {!isSorobanAsset && (
-                  <PillButton
-                    onClick={() => {
-                      dispatch(saveAsset(selectedAsset));
-                      navigateTo(ROUTES.sendPayment);
-                    }}
-                  >
-                    {t("SEND")}
-                  </PillButton>
-                )}
+                <PillButton
+                  onClick={() => {
+                    dispatch(saveAsset(selectedAsset));
+                    if (isContract) {
+                      dispatch(saveIsToken(true));
```

Comment:
@piyalbasu we really don't need to have `saveIsToken` since we can just check the issuer inline and avoid having to set this flag. I added a follow up task to refactor that away.